front/lib/models/assistant/conversation.ts

```diff
@@ -303,6 +307,13 @@ export class AgentMessageFeedback extends BaseModel<AgentMessageFeedback> {
 
   declare thumbDirection: AgentMessageFeedbackDirection;
   declare content: string | null;
+
+  declare agentMessage: NonAttribute<AgentMessage>;
+  declare user: NonAttribute<UserModel>;
+
+  // Make it easier to get the conversationId and messageId
+  declare conversationId: NonAttribute<Conversation["sId"] | undefined>;
```

Comment:
Sure about the NonAttribute there? We don't store that in the DB?

Reply:
Indeed, not needed because I follow the FKs to get the value, got rid of them